Unix/buildtool

```diff
@@ -966,7 +976,7 @@ if [ "$arg1" = "cshlibflags" -o "$arg1" = "cxxshlibflags" ]; then
     r=""
 
     case "$platform" in
-        LINUX_IX86_GNU|LINUX_X86_64_GNU|LINUX_PPC_GNU)
+        LINUX_IX86_GNU|LINUX_X86_64_GNU|LINUX_PPC_GNU|LINUX_ARM_GNU)
```

Comment:
These are duplicated so many times, I wonder if you'd be better off defining a GENERIC_LINUX symbol earlier and then just testing that. Then, in the future, if we need to add a platform, we add it in one place?